mix.exs

```diff
@@ -1,7 +1,7 @@
 defmodule Maestro.Mixfile do
   use Mix.Project
 
-  @version "0.1.1"
+  @version "0.1.2"
```

Comment:
Did this just not get updated or are you bumping the version in this PR?

Reply:
I see. I typically do a separate commit to bump the version which includes README changes and CHANGELOG updates and whatnot. This is fine though.